Client/dist/style.css

```diff
@@ -0,0 +1,35 @@
+.flex-column {
```

Comment:
For your css class names, personally i would try to be a little more specific to help you figure out which selector is for if you ever wanted to update your css